src/Compatibility/ControlGallery/src/Core/GalleryPages/TitleView.xaml

```diff
@@ -11,8 +11,8 @@
 	</NavigationPage.TitleView>
 	<ScrollView>
 		<StackLayout BackgroundColor="Pink">
-			<Button Clicked="masterDetailsPage_Clicked" Text="Flyout Details Page"></Button>
-			<Button Clicked="masterDetailsPageIcon_Clicked" Text="Toggle MDP Icon"></Button>
+			<Button Clicked="flyoutPagesPage_Clicked" Text="Flyout Details Page"></Button>
+			<Button Clicked="flyoutPagesPageIcon_Clicked" Text="Toggle MDP Icon"></Button>
```

Comment:
I see a couple of cases with `MDP` which should probably also go.

Reply:
We have alotttttt of them.. i think we keep it for now